test_pwnedpasswords.py

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env python3
 
 import unittest
-from . import pwnedpasswords
+import pwnedpasswords
 
 
 class TestPwnedPasswords(unittest.TestCase):
     def setUp(self):
         self.value = "123123"
-        self.num_matches = 2194818
+        self.num_matches = 11885347
```

Comment:
updated since the password list has grown significantly since this test was initially written